code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -322,7 +322,7 @@
 		to_chat(user, "<span class='notice'>You switch your helmet to travel mode. It will allow you to stand in zero pressure environments, at the cost of speed and armor.</span>")
 		name = "blood-red hardsuit helmet"
 		desc = "A dual-mode advanced helmet designed for work in special operations. It is in travel mode. Property of Gorlex Marauders."
-		flags = HEADCOVERSEYES | BLOCKHAIR | HEADCOVERSMOUTH | STOPSPRESSUREDMAGE | THICKMATERIAL | NODROP
+		flags = BLOCKHAIR | STOPSPRESSUREDMAGE | THICKMATERIAL | NODROP

```

Comment:
should the cover flags be toggled when this is turned on/off or am I missing something?